### ISSUE TITLE
Azusa wrong link

### ISSUE THREAD

**Issue by aliencaocao** [questianon/sdupdates#6]
> Blue Archives Azusa
> 
> Training set: 28 Input images, 20000 Steps, 5e-6:12000, 5e-7:30000 Learning rate. Training model: NAI-Full-Prunced Start with ba-azusa to get a good result. Recommend Hypernetwork Strength rate: 0.6 to 1.0 clip skip : 1
> 
> Link: https://cdn.discordapp.com/attachments/1032726084149583965/1033200762085453874/makotoshinkaiv2.pt
> Aesthetic Gradient: https://discord.com/channels/1015751613840883732/1033147620966801609/1033196208124088450

The link is pointing to makotoshinkaiv2
